src/github.ts

```diff
@@ -36,6 +36,10 @@ export const fetchMergedWithLabel = async (label: string) => {
       ),
     { headers: HEADERS },
   );
+  if (!response.ok) {
+    console.error(await response.text());
+    return;
```

Comment:
How does the caller code react to this returning `undefined`? Does it make sense to throw an error instead?

Reply:
The caller can't handle `undefined` but this is what happens anyway at the moment so I'm just trying to see why.
```
2024-04-29T08:48:16.944 app[32875639c95285] ams [info] error: Uncaught (in promise) TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))
2024-04-29T08:48:16.944 app[32875639c95285] ams [info] return Promise.all(
2024-04-29T08:48:16.944 app[32875639c95285] ams [info] ^
2024-04-29T08:48:16.944 app[32875639c95285] ams [info] at Function.all (<anonymous>)
2024-04-29T08:48:16.944 app[32875639c95285] ams [info] at file:///app/labels.ts:58:20
2024-04-29T08:48:16.944 app[32875639c95285] ams [info] at eventLoopTick (ext:core/01_core.js:168:7)
2024-04-29T08:48:16.944 app[32875639c95285] ams [info] at async Promise.all (index 0)
2024-04-29T08:48:16.944 app[32875639c95285] ams [info] at async Promise.all (index 0)
2024-04-29T08:48:16.944 app[32875639c95285] ams [info] at async Function.maintain (file:///app/labels.ts:25:3)
2024-04-29T08:48:17.505 app[32875639c95285] ams [info] INFO Main child exited normally with code: 1
```

I don't think we need to throw

Reply:
Ok fine as a debug measure, but I generally prefer to throw if such an error is unexpected and when the code can't possible continue when the request fails.

BTW, We should declare the return type of all functions in typescript later.